templates/common/on-prem/units/networkmanager.service.yaml

```diff
@@ -0,0 +1,6 @@
+name: NetworkManager.service
```

Comment:
this doesn't work, I'll investigate further. I see no change in NetworkManager.service.